mb/mb/hashes.py

```diff
@@ -210,7 +209,7 @@ def check_db(self, conn, verbose=False, dry_run=False, force=False):
                        self.hb.pgp or '',
                        self.hb.zblocksize,
                        self.hb.get_zparams(),
-                       binascii.hexlify(''.join(self.hb.zsums)),
+                       binascii.hexlify((''.join(self.hb.zsums)).encode('utf-8')).decode('utf-8'),
```

Comment:
uhm ... why?

Reply:
It doesn't compile in Python3 mixing str and byte types. Probably needs to be rewritten properly.